roles/network_plugin/flannel/templates/docker-network-options-systemd

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment="DOCKER_NETWORK_OPTIONS=--bip={{ flannel_subnet }} --mtu={{ flannel_mtu }}"
```

Comment:
I thought of yet another way to do this without breaking idempotency or upstart support. Will commit that soon.